src/js/adapter.js

```diff
@@ -181,7 +181,33 @@ if (navigator.mozGetUserMedia) {
 
   // The RTCPeerConnection object.
   RTCPeerConnection = function(pcConfig, pcConstraints) {
-    return new webkitRTCPeerConnection(pcConfig, pcConstraints);
+    var pc = new webkitRTCPeerConnection(pcConfig, pcConstraints);
+    var origGetStats = pc.getStats.bind(pc);
+    pc.getStats = function(selector, successCallback) {
```

Comment:
Hrm. That's a thought. It's going to ugly up the code a bit because we'll have to do type checking on the arguments to see if one is a function or not. Another option would be to keep the old function under a different name.